rails-commenting.rb

```diff
@@ -36,15 +36,15 @@ def destroy
     if @post.delete
       redirect_to blog_posts_path
     else
-      # 6)
+      # 6)method to redirect you to blog post create method rather than to staying on your deleted post...
       redirect_to blog_post_path(@post)
     end
   end
 
-  # 7)
+  # 7)adding strong params to manage exactly what goes into creating new object instances within your database
```

Comment:
and becusae of `private`, no method below this line can be called outside of the class